doc/Development-Guide-Xamarin.html

```diff
@@ -0,0 +1,554 @@
+﻿<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+
+<head>
+<meta content="text/html; charset=utf-8" http-equiv="Content-Type" />
+<link type="text/css" rel="stylesheet" href="bootstrap.min.css" />
+<title>Development Guide</title>
+<style type="text/css">
+ 
+
+</style>
+</head>
+
+<body>
+
+<h3>Introduction</h3>
+<p>Xamarin is a cross-platform UI toolkit that allows developers to efficiently 
+create native cross platform user interface layouts.<br>
+This document is intended to show how to setup a Xamarin development environment 
+and introduce existing features, user interfaces on Asp.Net Zero Xamarin platform.<br>
+For an overview of the Xamarin installation and setup practices see
+<a href="https://developer.xamarin.com/guides/xamarin-forms/getting-started/installation/">
+Xamarin.Forms Requirements</a> and
+<a href="https://developer.xamarin.com/guides/cross-platform/getting_started/installation/">
+Installation</a>. <br>
+For an overview of the Asp.Net Zero getting started see
+<a href="/Getting-Started-Core">Asp.Net Zero Getting Started.</a> The Asp.Net Zero 
+Xamarin is only available for Asp.Net Core! (Angular UI or Mvc UI)</p>
+<h4>Prerequisites</h4>
+<p>Following tools are needed in order to develop ASP.NET Zero Xamarin:</p>
```

Comment:
- Standardise `Asp.Net Zero` or `ASP.NET Zero`? ([some documents](https://github.com/aspnetzero/documents/pull/29/files#diff-168d6e075db031331a6ae09395f1ae97R69) use `AspNet Zero` too)
- But should standardise `ASP.NET Core` (e.g. [line 28](https://github.com/aspnetzero/documents/pull/29/files#diff-f1999aae9179dae277e5d40ab4bd3410R28))